static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -185,6 +185,7 @@ define([
             embedPath = $el.attr('data-embed-path'),
             // we need to look up the embedPath for main media videos
             canonicalUrl = $el.attr('data-canonical-url') || (embedPath ? '/' + embedPath : null),
+            shouldHideAdverts = $el.attr('data-block-video-ads') === 'false' ? false : true,
```

Comment:
Yea, so, look it up, if you can't find it, set to true.
With embeds we look it up though.